### PR TITLE
fix: Screenshot.defaults

### PR DESCRIPTION
**Pull request by Bkucera**
fix usage of `Cypress.Screenshot.defaults()`

- tests will fail since Cypress typedefs are incorrect

### CODE REVIEW

File: cypress/integration/examples/misc.spec.js [hunk 1]
```diff
@@ -89,8 +89,11 @@ context('Misc', () => {
         scale: false,
         disableTimersAndAnimations: true,
         screenshotOnRunFailure: true,
-        beforeScreenshot () { },
-        afterScreenshot () { },
+        // TODO: remove this when Cypress typedefs are fixed
+        // https://github.com/cypress-io/cypress/pull/7445
+        // @ts-ignore
+        onBeforeScreenshot () { },
+        onAfterScreenshot () { },
```
Comment (bahmutov) [inline cypress/integration/examples/misc.spec.js:96]: hmm, this is weird - it passes TS check with a single ts-ignore, not above each line?

Comment (Bkucera) [reply cypress/integration/examples/misc.spec.js:96]: it's strange, one ts-ignore seems to take care of both

**Comment by Bkucera** [comment]
how long are these ts-ignore comments going to be in here? we will need to remove them before releasing the next version on the test runner

**Comment by bahmutov** [comment]
We will update them right before the release

**Comment by Bkucera** [comment]
closing in favor of #425 since it has the correct branch name (so test-runner PR will use that for testing)

**Pull request closed without merging**